### ISSUE TITLE
api.php missing delete contact route

### ISSUE THREAD

**Issue by heinz-matt** [monicahq/monica#6621]
Was unable to delete contact with the api. Documentation gives this: DELETE /contacts/:id

looking at routes/api.php, found missing delete contact route. 
just needed to add the line below for it to work
Route::delete('/contacts/{contact}', 'ApiContactController@destroy');

**Comment by AQ-masatoshi-yamaguchi** [comment]
@heinz-matt 

Delete routing is already set up.
![image](https://user-images.githubusercontent.com/69567949/230726761-bae4d0b1-bf5a-4fd1-9c14-16f9279efa12.png)

Also, The following test also pass.
`php artisan test --filter it_deletes_a_contact`

**Comment by heinz-matt** [comment]
Thanks @AQ-masatoshi-yamaguchi for the reply. 
Sorry, for some reason that call wasn't working for me. I looked in Routes/api.php and didn't see a Delete for contacts, could be my complete ignorance of how the codebase works. Only after I added the Route::delete for contacts was I able to make calls to the api to delete contacts, so I figured it was just overlooked.

**Comment by JeaNugroho** [comment]
@heinz-matt If this is considered done, shall we close this issue? I thought it might be better to close issues that no longer needs to be worked on

**Comment by heinz-matt** [comment]
Yes, this can be closed if you believe the delete function is working as intended.
